src/main/java/com/sda/games/wheelOfFortune/db/ConnectionManager.java

```diff
@@ -0,0 +1,23 @@
+package com.sda.games.wheelOfFortune.db;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionManager {
+        String connectionString ="jdbc:hsqldb:file:src/main/java/com/sda/games/wheelOfFortune/db/database/dupa";
```

Comment:
RLY ? :P

Reply:
jak juz chcecie miec oddzielnabaze to zmiencie to na oddzielny katalog :P i wymyslcie lepsza nazwe np:
jdbc:hsqldb:file:src/main/java/com/sda/games/wheelOfFortune/db_wheelOfFortune/database"

Reply:
Sorry :)